client/client.go

```diff
@@ -26,6 +26,7 @@ import (
 	"strings"
 	"time"
 
+	openssl "github.com/Linutronix/golang-openssl"
```

Comment:
I guess we are also missing an update in the gomod files and `vendor/` folder...

Reply:
> I guess we are also missing an update in the gomod files and `vendor/` folder...

not really. Kristian have done that previously. so this is ready. mainly because we are already using openssl for signatures.